coresdk/src/coresdk/keyboard_input.cpp

```diff
@@ -68,21 +81,24 @@ namespace splashkit_lib
         }
     }
 
+
     void _keyboard_start_process_events()
     {
-        _key_pressed = false;
```

Comment:
Just curious as to why this was removed? To me it seems logical to ensure that `_key_pressed` is false before starting to process keyboard events. It should be false anyway as that's what it's initialized to but ensuring it's false shouldn't hurt

Reply:
_keyboard_start_process_events() is executed every time a process_events() call is made. The result of which is that the _key_pressed boolean is almost instantly toggled back to false. In other words, without that change if you called any_key_ressed() it would only return true for a fraction of a second, even if you were still holding down a key.

Reply:
Ah makes sense. Thanks.